src/sc/periodicals/configure.zcml

```diff
@@ -10,6 +9,7 @@
   <i18n:registerTranslations directory="locales" />
   <includeDependencies package="." />
```

Comment:
[other packages have the same directive](https://github.com/collective/collective.cover/blob/master/src/collective/cover/configure.zcml#L13), and if I remove it the tests fail.